### PR TITLE
iaq.md: add 2 entries

### PR DESCRIPTION
**Pull request by awokd**

### CODE REVIEW

**Comment by Aekez** [comment]
Looking good :) I really like the idea of this collection too, and that it keeps growing.

One thing I noticed, --refresh in dom0 if memory serves, was at a point broken (noticed in early period of Qubes 4, and noticed it worked again recently when working on the update script). So it seems like it works again now. I think --refresh in dom0 originally broke when the template update proxy was introduced in Qubes 4. So it's something like this for dom0, I think?

**Parts below is in "parts" speculation from observation, please be skeptical**
- **--clean (clears everything, pulls new metadata in full (no need for comparing metadata)).**
  - Disadvantage: Slow for user-client, and burden on server-agent.
  - Advantage: Clears potential bugs or errors in cache.
  - Advantage: May possibly be useful once in a while if cache becomes old and more error-prone.
- **--check-only (only checks updates with old metadata (speculation)).**
  - Disadvantage: Can be dangerous if metadata is indeed not renewed and a missmatch between templates and dom0 appears. But this is speculation on my part, I'm not sure if this is how it works.
  - Advantage: Fastest of them all, but not that much different to --refresh, although --refresh might be even slower, and just as slow as --clean, if a mismatch between metadata occurs (but that's a good thing, since it seems --check-only doesn't do that). But the speed, if viewed isolated, is still an advantage, though potentially faulty.
  - Assumptions are based on the quick speed of --check-only, the lack of timestamps on metadata that both --clean and --refresh provides, but --check-only does not, and also that the name of --check-only has ambiguity and can actually be misunderstood (it can also mean --check-only with old metadata, without renewing metadata, thereby potentially tricking the user to think it verifies new metadata, when it possibly doesn't). These 3 clues is what makes me believe that there is something wrong with --check-only, it might be better to improve it, or deprecate it entirely maybe?
- **--refresh (compares new and old metadata without downloading in full).**
  - Disadvantage: Doesn't clear old or buggy cache like --clean does.
  - Middle-ground: Faster than --clean, but slower than --check-only.
  - Advantage: Verifies metadata, similar to --clean, but apparently not done in --check-only.

So in short, --clean forces new metatadata, --check-only uses old metadata, and --refresh compares old and new metadata and downloads metadata if there is a difference? Is this how you guys see it too? Note that it's speculation whether --check-only only uses old metadata rather than neither comparing new/old (like --refresh), or downloading new metadata in full (like --clean does). But the time difference, and lack of new metadata timestamps in terminal, makes me suspicious of --check-only.

**Comment by Aekez** [comment]
oh, no no, your patch is perfectly fine and entirely fine to merge. Apologies if you thought anything was wrong with it, I'm purely bringing up another, but very related issue. Since we talked about a 2 days pending time for pull requests a while back, I was thinking it was a good opportunity to add a comment on something similar / related existing in the doc, that might be worth including too now that we're focusing on this area in this doc.

Whether to make a new pull request or not, I have no idea what to write in the doc in regards to the three commands in the first place, as I have too many assumptions about `--check-only` to clear out first, hence another reason why I bring it up for discussion rather than making a pull request and do the change request my self. Since this pull request is also general related to dom0 updates, I thought it was a good opportunity to add a comment about it here rather than making a "dummy" empty pull request to start a discussion on it, or start a new issue, since this pull request is highly related already. I don't mind a new issue or empty dummy pull request for that though, but I guess I should also have made my intention more clear when I wrote, I apologize for not being more clear in the first comment.

**Comment by awokd** [comment]
Sorry, thanks for the explanation! I expect there to be a certain amount of discussion if an entirely new document's being submitted, but this PR's pretty trivial. Maybe a new thread in admins would be best for discussing items not specifically related to a PR if there's not enough content for an issue.

**Comment by Aekez** [comment]
No problems :) I also didn't clarify probably when I should have. I'm fully ok with discussing it elsewhere, lets stick to the current PR change. Btw, just in case to clarify, I think everything looks good in the PR and see no issues with the current merge.
